### DISCUSSION TITLE
How do i use RoturTW

### DISCUSSION THREAD

**Discussion by Thebloxers998** [RoturTW/main#5] [General]
Like how

**Comment by ThatBeaverDev** [comment]
What platform are you trying to use it for?

**Reply by Thebloxers998** [reply]
Wdym

**Comment by ThatBeaverDev** [comment]
Are you trying to use it in scratch, javasript, python, where?
We have a repo called [main](https://github.com/RoturTW/main) that has all the platforms in a implementations folder, and under there all the languages that are supported. Just get it from there, and come back here if you have any questions

**Reply by Thebloxers998** [reply]
Scratch

**Reply by ThatBeaverDev** [reply]
Scratch or Turbowarp? Cos the folder name is misleading, it should say modded scratch, since they are the only ones that support custom extensions

**Reply by ThatBeaverDev** [reply]
You'll need to join the discord server to get an OS whitelisted anyways, so here's the link:

https://discord.gg/qkxskpGS2N

**Comment by ThatBeaverDev** [comment]
This should help

https://github.com/RoturTW/main/tree/main/Implementations/SCRATCH-MODS
